lms/static/js/spec/student_account/account_settings_spec.js

```diff
@@ -0,0 +1,42 @@
+define(['backbone', 'jquery', 'js/common_helpers/ajax_helpers', 'js/common_helpers/template_helpers',
```

Comment:
I would prefer to see tests added to this file that verify that the view as a whole is behaving correctly.